### PR TITLE
Cleanup ome-release repository

### PR DESCRIPTION
**Pull request by sbesson**
This PR:
- cleans all remaining instances of the Pelican module and the unnecessary `requirements.txt`
- fully rewrites the Readme to list all the currently supported pages and describe the new `make` commands used during  downloads page generation
- cleans some trivial items across the Python files (`hudson` -> `ci` ...)

### CODE REVIEW

**Comment by joshmoore** [comment]
Release builds for `5.0.2` all created with this. Need sign-off on README. Tomorrow at the latest? /cc @hflynn @mtbc Then I'll merge & tag.

File: README.md [hunk 1]
```diff
@@ -1,95 +1,78 @@
...
+===================
 
-These scripts are used during the OMERO release to copy the build
-artifacts to cvs.openmicroscopy.org and generate the
-[OMERO](http://www.openmicroscopy.org/site/products/omero/downloads) and
-[Bio-Formats](http://www.openmicroscopy.org/site/products/bio-formats/downloads)
-download pages for each release.
+These scripts are used during release to generate the downloads pages for each
+release of OME, i.e.:
+- [OMERO](http://downloads.openmicroscopy.org/omero) and
```
Comment (hflynn) [inline README.md]: Not a fan of the 'and' here, makes the bullet points rather unnecessary.

File: README.md [hunk 2]
```diff
@@ -1,95 +1,78 @@
...
+- [Bio-Formats](http://downloads.openmicroscopy.org/bio-formats).
 
-Prerequisites
--------------
+They are also used for each release of the following partner projects:
+- [FLIMFit](http://downloads.openmicroscopy.org/flimfit),
```
Comment (hflynn) [inline README.md]: These bullet points shouldn't really have punctuation at the end of them.

File: README.md [hunk 3]
```diff
@@ -1,95 +1,78 @@
...
-build numbers of the jobs used to produce the release artifacts.
+- for the OMERO.searcher downloads page generation, `PYSLIDVERSION` and
+  `RICERCAVERSION` determine the version numbers of the pyslid and ricerca
+  modules.
 
-- `VIRTUALBOX_PATH` and `ARTIFACT_PATH` are the folders where the OVA and the
-promoted builds are stored. Their default values are /ome/data_repo/virtualbox
-and /ome/data_repo/releases.
+- for OMERO, OMERO.figure, OMERO.searcher and OMERO.webtagging,
+  `ANNOUCEMENT_URL` specifies the URL of the announcement post to be used in
```
Comment (hflynn) [inline README.md:38]: 'ANNOUCEMENT_URL' is indeed what we have on the downloads page now I check but this is a typo I've missed before, both should be changed to ANNOUNCEMENT_URL.

File: README.md [hunk 4]
```diff
@@ -1,95 +1,78 @@
...
+- [OMERO.figure](http://downloads.openmicroscopy.org/figure),
+- [OMERO.mtools](http://downloads.openmicroscopy.org/mtools),
 
-* Merge all PRs related to the upcoming release. Update all submodules using the OMERO-submods-{stable, develop} jobs.
+and some custom third-party packages:
+- [Ice](http://downloads.openmicroscopy.org/ice).
 
-* Tag the `RELEASE` of openmicroscopy.git and bioformats.git
+Prior to the downloads page generation, the artifacts for the corresponding
+release must have been copied under the root folder of the downloads page
```
Comment (mtbc) [inline README.md]: Does this mean "copied to"?

File: README.md [hunk 5]
```diff
@@ -1,95 +1,78 @@
...
-OMERO and Bio-Formats artifacts are symlinked and downloadable. Their default
-values are /var/www/cvs.openmicroscopy.org.uk/snapshots and
-http://cvs.openmicroscopy.org.uk/snapshots.
+Additional environment variables are defined in the scripts but should not
+need to be modified:
 
-- `ANNOUCEMENT_URL` is the URL of the announcement post to be used in the
-OMERO downloads page.
+- `RSYNC_PATH` and `PREFIX` are the root folder and the prefix of the folder
+  where the release artifacts and downloads page.
```
Comment (mtbc) [inline README.md]: huh?

File: README.md (hunk 3 above)
Comment (sbesson) [inline README.md:38]: Agreed. Not as part of this PR though. I can do that in another PR and update the release envvars.

**Comment by sbesson** [comment]
Fixes pushed. As i said above, the point about `ANNOUNCEMENT_URL` is valid but does not belong to this tidying PR as it requires coordination with the release jobs.

File: README.md (hunk 3 above)
Comment (hflynn) [inline README.md:38]: Fair enough, sorry I missed it before.

File: README.md [hunk 6]
```diff
@@ -1,95 +1,78 @@
...
 
-These scripts are used during the OMERO release to copy the build
-artifacts to cvs.openmicroscopy.org and generate the
-[OMERO](http://www.openmicroscopy.org/site/products/omero/downloads) and
-[Bio-Formats](http://www.openmicroscopy.org/site/products/bio-formats/downloads)
-download pages for each release.
+These scripts are used during release to generate the downloads pages for each
+release of OME, i.e.:
+- [OMERO](http://downloads.openmicroscopy.org/omero)
+- [Bio-Formats](http://downloads.openmicroscopy.org/bio-formats).
```
Comment (hflynn) [inline README.md]: Still punctuation here

File: README.md [hunk 7]
```diff
@@ -1,95 +1,78 @@
...
+- [FLIMFit](http://downloads.openmicroscopy.org/flimfit)
+- [u-track](http://downloads.openmicroscopy.org/u-track)
+- [OMERO.webtagging](http://downloads.openmicroscopy.org/webtagging)
+- [OMERO.searcher](http://downloads.openmicroscopy.org/searcher)
+- [OMERO.figure](http://downloads.openmicroscopy.org/figure)
+- [OMERO.mtools](http://downloads.openmicroscopy.org/mtools)
 
-* Merge all PRs related to the upcoming release. Update all submodules using the OMERO-submods-{stable, develop} jobs.
+and some custom third-party packages:
+- [Ice](http://downloads.openmicroscopy.org/ice).
```
Comment (hflynn) [inline README.md]: and here.

**Comment by sbesson** [comment]
Pushed

**Comment by hflynn** [comment]
Thank you! I'm happy now :) Will merge once Travis is green

**Comment by joshmoore** [comment]
Beat ya to it! Merging & Tagging.

**Comment by sbesson** [comment]
--rebased-to #113

**Pull request merged**